packages/ui/src/components/Dialog/LargeDialog.css.ts

```diff
@@ -0,0 +1,40 @@
+import { style } from '@vanilla-extract/css'
+import { minWidth, tokens } from '../../theme'
+
+export const closeButton = style({
+  position: 'absolute',
+  right: tokens.space.md,
+  top: tokens.space.md,
+})
+
+export const content = style({
+  display: 'flex',
+  width: '100%',
+})
+
+export const card = style({
+  position: 'relative',
+  marginBlock: 'auto',
+  marginInline: tokens.space.md,
+  width: '100%',
+  maxHeight: `calc(100vh - (${tokens.space.md}) * 2)`,
+  // If card has header inside, then other content should generally have its own wrapper with overflowY: 'auto'
+  // to make header and close button sticky. This is a fallback for dialogs which don't have a header
+  overflowY: 'auto',
+
+  '@media': {
+    [minWidth.sm]: {
+      marginBlock: 'auto',
+      marginInline: 'auto',
+      maxWidth: 'min(60rem, 80%)',
+    },
```

Comment:
It has larger padding-right in desktop to make room for close button
```suggestion
    },
    [minWidth.lg]: {
      paddingRight: tokens.space.xxl',
    },
```

Reply:
I assumed only header need padding. This has its pros and cons, will talk to Nils before making a decision

Reply:
Nils approved having padding just in the header, it looks better overall even if header-less content may in rare cases go under the button